cocos/base/CCDirector.cpp

```diff
@@ -260,7 +254,7 @@ void Director::setGLDefaultValues()
     setProjection(_projection);
 
     // set other opengl default values
-    glClearColor(_clearColor.r, _clearColor.g, _clearColor.b, _clearColor.a);
+    setClearColor(Color4F::BLACK);
```

Comment:
If that's the case, it's probably safe to remove it since it just re-apply cached color, and renderer already set default color as black.